### ISSUE TITLE
Can you test my rule pack?

### ISSUE THREAD

**Issue by AgelxNash** [NotSoSecure/password_cracking_rules#2]
https://gist.github.com/AgelxNash/e7215886776e27dd023b75064756437e

**Comment by stealthsploit** [comment]
It's a slightly subjective question, but I tested it [against the same data](https://www.notsosecure.com/one-rule-to-rule-them-all/) that I used to create OneRuleToRuleThemAll in the first place, so that it was comparatively consistent.

The output from your rules using your rule was:

Session..........: hashcat
Status...........: Exhausted
Hash.Type........: MD5
Hash.Target......: b:\Leaks\lifeboat-raw
Time.Started.....: Wed Oct 04 19:30:14 2017 (2 hours, 56 mins)
Time.Estimated...: Wed Oct 04 22:26:25 2017 (0 secs)
Guess.Base.......: File (b:\Dictionaries\rockyou.txt)
Guess.Mod........: Rules (rules\AgelxNash_Rules.txt)
Guess.Queue......: 1/1 (100.00%)
Speed.Dev.#2.....:   834.7 MH/s (113.81ms)
Recovered........: 2779322/4331053 (64.17%) Digests, 0/1 (0.00%) Salts
Recovered/Time...: CUR:5781,806446,N/A AVG:15773,946401,22713634 (Min,Hour,Day)
Progress.........: 745679272448/745679272448 (100.00%)
Rejected.........: 104287928/745679272448 (0.01%)
Restore.Point....: 14343296/14343296 (100.00%)


In comparison to the original tests:
OneRuleToRuleThemAll.rule:   2,960,711 cracked (68.36%)
anRules.rule:                            2,779,322 cracked (64.17%)

All in all it's a decent rule set and worked well. Good job!
